src/hooks/useSearchResults.js

```diff
@@ -7,93 +7,122 @@ const useSearchResults = () => {
 
   // ... (Logic for displaySearchResults)
   async function displaySearchResults(searchTerm) {
-    const url = 'https://en.wikipedia.org/w/api.php?';
+    const url = "https://en.wikipedia.org/w/api.php?";
     let params = {
-      action: 'query',
-      list: 'search',
-      format: 'json',
-      origin: '*',
+      action: "query",
```

Comment:
did this happen automatically through a tool you used or did you do it manually? cant wait to hear input on that!

Reply:
Also curious about this